cargo-near/src/lib.rs

```diff
@@ -1,136 +0,0 @@
-use std::{env, str::FromStr};
```

Comment:
lib.rs should stay as we want to make use of cargo-near commands in near-workspaces-rs, similarly to how we use near-cli-rs here. Please, move most of things except main() function back from main.rs to here.

Reply:
Done